docker/ci/Dockerfile

```diff
@@ -70,7 +70,9 @@ ENV BUNDLE_WITHOUT="development:production:docker"
 
 RUN --mount=type=cache,target=/app/frontend/.angular/cache,uid=1000,gid=1000 DATABASE_URL=nulldb://db bin/rails openproject:plugins:register_frontend assets:precompile && \
 	# only keep most current angular cache since webpack is unable to cleanup after itself
-	find frontend/.angular/cache -type d -exec sh -c 'ls -dt "$1"/*/ | tail -n +2 | xargs rm -r' sh {} \;
+  ls -1 --directory --sort=time frontend/.angular/cache/*/angular-webpack/* \
```

Comment:
I didn't test this; but you might be able to get away with just `find` if you have busy box tools at your disposal.
something like...
```sh
find frontend/.angular/cache/*/angular-webpack/* -mindepth 1 -maxdepth 1 -type d -exec rm -r {} +
```

Reply:
Thanks for the suggestion but that would not produce the expected result.
The goal is to delete all folders except the most recent one. The command you're suggesting would delete all folders.